ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -187,6 +187,19 @@ switch:
                           switch.default_evpn_role, 'none') %}
   evpn_role: {{ switch_evpn_role }}
 
+{# switch.evpn_gateway #}
+{%     set switch_evpn_gateway = switch_data.combined.evpn_gateway.evpn_l2.enabled is arista.avd.defined(true) or
+                                 switch_data.combined.evpn_gateway.evpn_l3.enabled is arista.avd.defined(true) or
+                                 switch_data.combined.evpn_gateway.vpn_ipv4.enabled is arista.avd.defined(true) %}
```

Comment:
This knob wasn't mentioned in the fabric-topology.md documentation or in your PR description. Just making sure this wasn't added by mistake.

Reply:
Just reviewed. Probably inherited from a point in time of the code. I see this variable is only used in this file and I am not really sure it will be needed when MPLS is added. Probably a good idea to refactor or leave like that as it is easier to read. Comments?

Reply:
Still there...